website/static/files/build_orca_inventory_refined.sql

```diff
@@ -85,6 +86,8 @@ BEGIN
       IS 'Step function execution ID from AWS';
     COMMENT ON COLUMN granules.ingest_time
       IS 'Date and time the granule was originally ingested into ORCA.';
+    COMMENT ON COLUMN granules.cumulus_create_time
+      IS 'Cumulus timestamp as part of the sync granule task output';
```

Comment:
Change comment to match what is in the orca_sql.py for this column.

Reply:
fixed